### DISCUSSION TITLE
[Feature] Support for rendering Ultra HDR images like in Google Photos...

### DISCUSSION THREAD

**Discussion by FibreTTP** [immich-app/immich#5988] [Feature Request]
### The feature

...where basically, the screen brightness of the bright parts of the image is turned up. I don't exactly know how it works, but it was brought up in a discussion of a [similar, but different feature request](https://github.com/immich-app/immich/discussions/3623#discussioncomment-7313883) which I will directly quote here:

> "Now that Android 14 is introducing "Ultra HDR" which is just a gain map on top of a standard P3 JPEG it might be worthwhile including HDR image support now too. At this time though there doesn't seem to be an easy way to use ffmpeg or https://github.com/ImageMagick/ImageMagick/discussions/5441 to do this unfortunately. It always seems to clip the values when converting formats or downscaling in my experience. It would seem to me that the gain map support would be more straightforward to include at least in Android 14. I would be shocked if Apple was fast to adopt" - @AngelaDMerkel

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by benmccann** [comment]
Closing as duplicate of https://github.com/immich-app/immich/discussions/7262. While this one was filed first, that one has attracted more attention and has more details
